config.libsonnet

```diff
@@ -6,6 +6,9 @@
     kubeStateMetricsSelector: 'job="default/kube-state-metrics"',
     nodeExporterSelector: 'job="default/node-exporter"',
     notKubeDnsSelector: 'job!="kube-system/kube-dns"',
+    kubeSchedulerSelector: 'job="kube-scheduler"',
+    kubeControllerManagerSelector: 'job="kube-controller-manager"',
```

Comment:
On GKE we can't scrape these two components (AFAICT).  So we need a way of disabling those alerts.